generic_config_updater/gu_common.py

```diff
@@ -184,17 +184,18 @@ def _invoke_validating_function(cmd):
 
         for element in patch:
             path = element["path"]
+            operation = element["op"]
             match = re.search(r'\/([^\/]+)(\/|$)', path) # This matches the table name in the path, eg if path if /PFC_WD/GLOBAL, the match would be PFC_WD
             if match is not None:
-                table = match.group(1)
+                table = match.group(1).lower()
```

Comment:
>lower() [](http://example.com/codeflow?start=39&length=7)

Why change to lowercases?

Reply:
I changed it back to default text which would be upper case, and also changed the config file we are matching against to have table names in upper case